src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -144,6 +146,22 @@
                                               :value (get (:attributes @state) property-kwd)
                                               :onChange update-property
                                               :rows 3})
+                     (= typeahead "ontology")
+                     [:div {}
+                      (style/create-text-field {:ref property-kwd
+                                                :className "typeahead"
+                                                :placeholder "Select an ontology value."
+                                                :style {:width "100%" :marginBottom "0px"}})
+                      (let [relatedID (library-utils/get-related-id (:attributes @state) library-schema property-kwd
+                                                                      (library-utils/get-related-id (:attributes props) library-schema property-kwd nil))
```

Comment:
this feels a little convoluted ... it's looking for the related id in state, and falls back to props if it's not in state? Is this necessary? I'm not quite following the logic, so I leave it up to you.